README.md

```diff
@@ -183,6 +183,20 @@ config :metrix,
   context: %{"source" => "my-app"}
 ```
 
+An optional (global) prefix may also be configured:
+
+```elixir
+config :metrix,
+  metric_prefix: "my-prefix."
```

Comment:
Can we remove the `metric_` part of the name here (and throughout) to be consistent w/ how we're handling the global `context`?